packages/shopify-codemod/test/transforms/remove-return-voids.js

```diff
@@ -0,0 +1,14 @@
+import 'test-helper';
+import removeReturnVoids from 'remove-return-voids';
+
+describe('removeReturnVoids', () => {
+  it('Removes void returns from else generated by Decaf', () => {
+    expect(removeReturnVoids).to.transform('remove-return-voids/void');
+  });
+  it('Removes undefined returns from else generated by Decaf', () => {
+    expect(removeReturnVoids).to.transform('remove-return-voids/undefined');
+  });
+  it('Doesn\'t remove valid returns', () => {
+    expect(removeReturnVoids).to.transform('remove-return-voids/valid');
+  });
+});
```

Comment:
Need a check to make sure that this only affects conditions that are the last expression in the function.